internal/model/trip.go

```diff
@@ -6,11 +6,12 @@ import (
 
 // TripTemplate represents a saved trip template.
 type TripTemplate struct {
-	Name        string `json:"name"`
-	Origin      string `json:"origin"`
-	Destination string `json:"destination"`
-	TripType    string `json:"tripType"`
-	Notes       string `json:"notes"`
+	Name          string `json:"name"`
+	Origin        string `json:"origin"`
+	Destination   string `json:"destination"`
+	TripType      string `json:"tripType"`
+	ActivityType  string `json:ActivityType` //←add ActivityType
```

Comment:
Let's keep the camelCase here for consistency so that it matches the other strings.  Also added the missing double quotes.

```suggestion
	ActivityType  string `json:"activityType"` 
```

Can you also update the `trip_test.go` with the appropriate test case so that we can test for the new activity type validation?